[2025-04-02T20:10:59.564+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Weather_Proj.Write_data_in_kafka scheduled__2025-04-02T19:20:00+00:00 [queued]>
[2025-04-02T20:10:59.572+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Weather_Proj.Write_data_in_kafka scheduled__2025-04-02T19:20:00+00:00 [queued]>
[2025-04-02T20:10:59.572+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-04-02T20:10:59.587+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): Write_data_in_kafka> on 2025-04-02 19:20:00+00:00
[2025-04-02T20:10:59.595+0000] {standard_task_runner.py:60} INFO - Started process 2514 to run task
[2025-04-02T20:10:59.595+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2025-04-02T20:10:59.600+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Weather_Proj', 'Write_data_in_kafka', 'scheduled__2025-04-02T19:20:00+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/Project_tasks.py', '--cfg-path', '/tmp/tmpvhhllww2']
[2025-04-02T20:10:59.601+0000] {standard_task_runner.py:88} INFO - Job 21: Subtask Write_data_in_kafka
[2025-04-02T20:10:59.653+0000] {task_command.py:423} INFO - Running <TaskInstance: Weather_Proj.Write_data_in_kafka scheduled__2025-04-02T19:20:00+00:00 [running]> on host 0d0dab2febfc
[2025-04-02T20:10:59.735+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Weather_Proj' AIRFLOW_CTX_TASK_ID='Write_data_in_kafka' AIRFLOW_CTX_EXECUTION_DATE='2025-04-02T19:20:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-02T19:20:00+00:00'
[2025-04-02T20:12:45.706+0000] {local_task_job_runner.py:302} WARNING - State of this instance has been externally set to upstream_failed. Terminating instance.
[2025-04-02T20:12:45.707+0000] {process_utils.py:131} INFO - Sending 15 to group 2514. PIDs of all processes in the group: [2514]
[2025-04-02T20:12:45.708+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 2514
[2025-04-02T20:13:45.710+0000] {process_utils.py:149} WARNING - process psutil.Process(pid=2514, name='airflow task ru', status='sleeping', started='20:10:58') did not respond to SIGTERM. Trying SIGKILL
[2025-04-02T20:13:45.713+0000] {process_utils.py:86} INFO - Sending the signal 9 to group 2514
[2025-04-02T20:13:45.727+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=2514, name='airflow task ru', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='20:10:58') (2514) terminated with exit code -9
[2025-04-02T20:13:45.727+0000] {standard_task_runner.py:175} ERROR - Job 21 was killed before it finished (likely due to running out of memory)
[2025-04-02T20:35:49.672+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Weather_Proj.Write_data_in_kafka scheduled__2025-04-02T19:20:00+00:00 [queued]>
[2025-04-02T20:35:49.680+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Weather_Proj.Write_data_in_kafka scheduled__2025-04-02T19:20:00+00:00 [queued]>
[2025-04-02T20:35:49.681+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-04-02T20:35:49.692+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): Write_data_in_kafka> on 2025-04-02 19:20:00+00:00
[2025-04-02T20:35:49.696+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2025-04-02T20:35:49.696+0000] {standard_task_runner.py:60} INFO - Started process 1589 to run task
[2025-04-02T20:35:49.701+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Weather_Proj', 'Write_data_in_kafka', 'scheduled__2025-04-02T19:20:00+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/Project_tasks.py', '--cfg-path', '/tmp/tmp0wr0b68t']
[2025-04-02T20:35:49.702+0000] {standard_task_runner.py:88} INFO - Job 26: Subtask Write_data_in_kafka
[2025-04-02T20:35:49.743+0000] {task_command.py:423} INFO - Running <TaskInstance: Weather_Proj.Write_data_in_kafka scheduled__2025-04-02T19:20:00+00:00 [running]> on host 34d96960033d
[2025-04-02T20:35:49.805+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Weather_Proj' AIRFLOW_CTX_TASK_ID='Write_data_in_kafka' AIRFLOW_CTX_EXECUTION_DATE='2025-04-02T19:20:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-02T19:20:00+00:00'
[2025-04-02T20:39:31.952+0000] {local_task_job_runner.py:302} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2025-04-02T20:39:31.955+0000] {process_utils.py:131} INFO - Sending 15 to group 1589. PIDs of all processes in the group: [1589]
[2025-04-02T20:39:31.956+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 1589
[2025-04-02T20:40:32.004+0000] {process_utils.py:149} WARNING - process psutil.Process(pid=1589, name='airflow task ru', status='sleeping', started='20:35:48') did not respond to SIGTERM. Trying SIGKILL
[2025-04-02T20:40:32.081+0000] {process_utils.py:86} INFO - Sending the signal 9 to group 1589
[2025-04-02T20:40:32.147+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=1589, name='airflow task ru', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='20:35:48') (1589) terminated with exit code -9
[2025-04-02T20:40:32.160+0000] {standard_task_runner.py:175} ERROR - Job 26 was killed before it finished (likely due to running out of memory)
